scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -814,16 +814,32 @@ let notUsingGitPush1by1 =
 
         let json = GitHubApiCall url
 
-        json.Contains "\"check_suites\":[]"
+        not(json.Contains "\"check_suites\":[]")
     )
-    |> Seq.contains true
 
-if notUsingGitPush1by1 then
+let notActivatedGitHubActions = not(Seq.contains true hasCiStatus)
```

Comment:
@realmarv let's rename this to gitHubActionsEnabled (and then remove the `not`), so that...